ocp_cleanup.sh

```diff
@@ -49,6 +49,12 @@ for vm in $(sudo virsh list --all --name | grep "^${CLUSTER_NAME}.*bootstrap");
   sudo virsh destroy $vm
   sudo virsh undefine $vm --remove-all-storage
 done
+
+# For some reason --remove-all-storage doesn't actually remove the storage...
+if [ -d /var/lib/libvirt/openshift-images ]; then
+  sudo rm -f /var/lib/libvirt/openshift-images/${CLUSTER_NAME}-*
+fi
+
```

Comment:
When https://github.com/openshift-metal3/dev-scripts/pull/1146 lands we can perhaps do something smarter since the pool isn't default from 4.6 onwards so the logic below could handle this?

Reply:
In my dsal machine I see
```
/var/lib/libvirt/openshift-images/ostest-nj5l9-bootstrap/ostest-nj5l9-bootstrap
```
Doesn't that mean that we should have rm work recursively?

Reply:
Yes, good point - thanks!